ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywallView.kt

```diff
@@ -60,6 +61,10 @@ private fun getPaywallViewModel(
             offering,
             listener,
             MaterialTheme.colorScheme,
+            preview = isInPreviewMode(),
         ),
     )
 }
+
+@Composable
```

Comment:
I think you can add a `@ReadOnlyComposable` annotation here for some small optimizations.